### PR TITLE
Split packaging

### PR DESCRIPTION
**Pull request by om26er**
* Split code into two different packages: `autobahn` and `autobahn-xbr`

Note: `autobahn-xbr` does not have a dependency on `autobahn` in terms of imports (require stanza).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "autobahn",
-  "version": "19.7.2",
+  "version": "19.7.3",
   "description": "An implementation of The Web Application Messaging Protocol (WAMP).",
   "main": "index.js",
+  "private": true,
```
Comment (om26er) [inline package.json]: This `package.json` is only for local development and CI usage. `"private": true"` makes it safe from accidental `npm publish`.

File: packages/autobahn/package.json [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  },
+  "dependencies": {
+    "cbor": ">= 3.0.0",
+    "crypto-js": ">=3.1.8",
+    "msgpack5": ">= 3.6.0",
+    "tweetnacl": ">= 0.14.3",
+    "tweetnacl-sealedbox-js": ">=1.1.0",
+    "when": ">= 3.7.7",
+    "ws": ">= 1.1.4",
+    "sha3": ">=2.0.1",
```
Comment (kuema) [inline packages/autobahn/package.json]: I think this dependency should be moved to `autobahn-xbr` as well. It was added together with the xbr feature in #433.

File: package.json (hunk 1 above)
Comment (oberstet) [reply package.json]: Neat .. wasn't aware of that one ..

**Comment by oberstet** [comment]
so for me, this looks good for merging (modulo above comments)!

File: packages/autobahn/package.json (hunk 2 above)
Comment (om26er) [reply packages/autobahn/package.json]: Moved `sha3`, `uuid` and `tweetnacl-sealedbox-js` to autobahn-xbr package.

**Pull request merged**